src/app/components/autocomplete/autocomplete.ts

```diff
@@ -496,6 +498,9 @@ export class AutoComplete implements AfterViewChecked,AfterContentInit,OnDestroy
 
             if (!this.overlay.style.minWidth) {
                 this.overlay.style.minWidth = DomHandler.getWidth(this.el.nativeElement.children[0]) + 'px';
+                if (this.overlayFitWidth) {
+                    this.overlay.style.maxWidth = this.overlay.style.minWidth;
```

Comment:
I wasn't really sure if I should have used simply `width` instead of a combination of both `minWidth` and `maxWidth`. But due to CSS weirdness sometimes, I opted for the latter.